packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -71,17 +74,29 @@ describe('background sync queue test', () => {
 
   it('check push proxy', async () => {
     const currentLen = backgroundSyncQueue._queue.queue.length;
-    await backgroundSyncQueue.pushIntoQueue({request: new Request('http://lipsum.com')});
+    await backgroundSyncQueue.pushIntoQueue({request: new Request('http://localhost:3000/__echo/counter')});
+    await backgroundSyncQueue.replayRequests();
     chai.assert.equal(backgroundSyncQueue._queue.queue.length, currentLen + 1);
```

Comment:
The 'currentLen + 1' always concerned me a bit here, is this test reliant on other tests passing (i.e. if one test is incorrect and the length is wrong or currentLen is not updated, will this test fail?